README.md

```diff
@@ -1,29 +1,33 @@
-`wolverine --> ⓦ0lv3r1nⓔ`
-## I am alive
-https://cool-name-api.glitch.me/
+This is a website and node.js project to transform text into a cool representation of that text.
 
+For example `wolverine` becomes:
 
-## Build Setup
+* `ⓦ0lv3r1nⓔ`
+* `wo|ver!ne`
+* `🅆🄾🄻🅅🄴🅁🄸🄽🄴`
+* `ⓦⓞⓛⓥⓔⓡⓘⓝⓔ`
+*  (backwards) `‮wolverine`
+
+You can visit the website at https://cool-name-api.glitch.me/
+
+and make JSON API requests to https://cool-name-api.glitch.me/coolify?name=wolverine.
+
+# Build Setup
+
+This is a node.js project. With Node installed you can start the listening webserver with
 
 ```
 npm start
 ```
-# Author
```

Comment:
Don't you think we should at last keep some of this? 😅

Reply:
I don’t know. The info is not central to the project so I removed it [for now, because the intent was unclear to me]. I guess you want to be mentioned like that/there to promote your persona?

Do you want to list all contributors? Main contributors?

Maybe it should be a list with smaller link icons?

Or we can just keep that part as it was. Feel free to revert this Author section and merge the rest.

Reply:
>Do you want to list all contributors? Main contributors?

That would be great.

>Maybe it should be a list with smaller link icons?

Agreed.

Reply:
Do you want me to do something in that regard? Before landing this PR/in this PR?

Or do you want to land this as it is, and then work on a contributors section in the README?

Reply:
I guess 2nd one is better, we can add contributors later.